grpc_gateway/deps.bzl

```diff
@@ -9,6 +9,10 @@ filegroup(
 )
 """
 
+# use a version that uses rules_go 0.12+. unfortunately, since the last tagged release came in may,
+# we have to reference HEAD.
+com_github_grpc_ecosystem_grpc_gateway_commit="bb916ca4f209e5f2a004537658d944ef4ae4409e" # Aug 7, 2018
```

Comment:
Do we know what we rely on this package for?

Reply:
we don't, but it's used as part of the test suite in the repo. and since it requires rules_go to build, so we can't have the build be green without upgrading this.